OSDOCS-17110 CQA for Microshift (#106151)
openshift/openshift-docs: modules/microshift-install-rpm-before.adoc

[id="microshift-install-rpm-before_{context}"]
= Before installing {microshift-short} from an RPM package

[role="_abstract"]

> Preparation of the host machine is recommended before installing {microshift-short} for memory configuration and FIPS mode.

Reviewer: ```suggestion
Before installing {microshift-short} for memory configuration and FIPS mode, you must prepare the host.
```
better

Author: Done
